tools/fast/process_stderr_recorder_test.go

```diff
@@ -9,10 +9,13 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
 	mockplugin "github.com/filecoin-project/go-filecoin/tools/iptb-plugins/filecoin/mock"
 )
 
 func TestStartLogCapture(t *testing.T) {
+	tf.IntegrationTest(t)
```

Comment:
I don't know if I'd call any of these FAST tests integration tests. They are unit tests for FAST. They use a mock plugin which doesn't actually start any external processes, etc.

Reply:
At least they shouldn't be bundled with the daemon tests.

Reply:
nm, lets leave this as is. If it becomes a problem we can figure it out later.

Reply:
I didn't read into these test too far when making the changes, if these don't fork I think they should be unit tests, thoughts?

Reply:
I have previously implied that forking automatically excludes something from being a unit test, but that is too harsh. If the forked process does almost no work and/or communication and the whole test completes in 10s of milliseconds then it can remain a unit test. But if the forked process does work like initialising a node, it's probably an integration test. It's a judgement call.